tests/5.1/order/test_taskloop_simd_order_unconstrained_device.c

```diff
@@ -0,0 +1,56 @@
+//===------------ test_target_taskloop_simd_order_reproducible_device.c ------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test checks that the order(reproducible:unconstrained) clause is properly handled.
+// As per the definition the order in which they are executed does not matter. 
+// Instead focus on that the correct result is calculated, regardless of execution
+// order. 
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 24
+
+int main() {
+	int errors = 0;
+	int x[N];
+	int y[N];
+
+	for (int i = 0; i < N; i++) {
+		x[i] = i;
+		y[i] = i;
+	}
+
+	OMPVV_TEST_OFFLOADING;
+
+	#pragma omp target map(tofrom: x,y)
+	{
+		#pragma omp parallel
+		{
+		   #pragma omp single
+		   #pragma omp taskloop simd order(reproducible:unconstrained)
```

Comment:
GCC fails with the slightly misleading error message:
```
test_taskloop_simd_order_unconstrained_device.c:36:65: error: expected ‘concurrent’ before ‘unconstrained’
   36 |                    #pragma omp taskloop simd order(reproducible:unconstrained)
```

However, it is rightly complains. OpenMP 5.1 has:
> The syntax of the order clause is as follows:
> `order([ order-modifier :] concurrent)`
> where _order-modifier_ is one of the following:
> `reproducible`
> `unconstrained`

Thus, the order clause argument must end with `concurrent`. I assume, you meant:

```suggestion
		   #pragma omp taskloop simd order(unconstrained:concurrent)
```

Reply:
Oh haha I flipped them, thanks for the catch @tob2!